### ISSUE TITLE
Dependency `request` uses a vulnerable version of `cryptiles`

### ISSUE THREAD

**Issue by Puzzleton** [auth0/node-jwks-rsa#83]
```
- jwks-rsa@^1.4.0
  - request@^2.73.0
    - hawk 3.1.3
      - cryptiles 2.x.x
```

`cryptiles` has a known [high priority vulnerability](https://nvd.nist.gov/vuln/detail/CVE-2018-1000620). The latest version of `request` (v2.88.0) no longer depends on `hawk`, thus removing the vulnerability with `cryptiles`.

**Comment by danwkennedy** [comment]
Similarly, it looks like there's a vulnerability in the `extend` package that `request@2.88.0` fixes:

**Comment by damieng** [comment]
Fixed in #91 - nothing exists yet for the extend package/beyond 2.88.  Might be possible to switch to something other than 'request' in a future version.
